drivers/espi/Kconfig.it8xxx2

```
@@ -21,6 +21,81 @@ config ESPI_PERIPHERAL_HOST_IO
 config ESPI_PERIPHERAL_DEBUG_PORT_80
 	default y
 
+config ESPI_PERIPHERAL_EC_HOST_CMD
+	default y
+
+choice IT8XXX2_H2RAM_HC_SIZE_CHOICE
+	prompt "choose a H2RAM space for ec host command"
```

Just:
```suggestion
	prompt "H2RAM space for ec host command"
```

as per https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

and looks cleaner in `menuconfig` also.

---

Done.